src/osd/PGBackend.h

```diff
@@ -299,6 +299,7 @@ typedef std::shared_ptr<const OSDMap> OSDMapRef;
      virtual bool check_failsafe_full() = 0;
 
      virtual void inc_osd_stat_repaired() = 0;
+     virtual void set_osd_stat_repaired(int64_t) = 0;
```

Comment:
Why is this needed? unlike inc_osd_stat_repaired(), there is no clearing from the backend